Source/AlchemyDataNewsV1/Tests/AlchemyDataNewsV1Tests.swift

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+import XCTest
+import AlchemyDataNewsV1
+
+class AlchemyDataNewsV1Tests: XCTestCase {
+    
+    private var service: AlchemyDataNewsV1!
+
+    private let timeout: NSTimeInterval = 120.0
+    
+    override func setUp() {
+        super.setUp()
+        if let url = NSBundle(forClass: self.dynamicType).pathForResource("Credentials", ofType: "plist") {
+            if let dict = NSDictionary(contentsOfFile: url) as? Dictionary<String, String> {
+                let apikey = dict["AlchemyAPIKey"]!
+                if service == nil {
```

Comment:
There's absolutely nothing wrong with the correctness of this statement, but I just want to provide some explanation.

In the past, we used `if service == nil { service = ... }` thinking that the same `service` could be used for multiple tests. So we updated a lot of the tests to use that `if service == nil { ... }` pattern. But then we found out that the entire class is re-instantiated for each test. So that code isn't helpful--`service` is always `nil`.

Anyway, there's not really any need to change this. But I just wanted to share what we learned in the past.